products/apache-camel.md

```diff
@@ -0,0 +1,218 @@
+    latest: "3.10.0"
+    latestReleaseDate: 2021-05-15
+
+-   releaseCycle: "3.9"
+    releaseDate: 2021-03-22
+    eol: 2021-03-22
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.9.0"
+    latestReleaseDate: 2021-03-22
+
+-   releaseCycle: "3.8"
+    releaseDate: 2021-02-09
+    eol: 2021-02-09
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.8.0"
+    latestReleaseDate: 2021-02-09
+
+-   releaseCycle: "3.7"
+    releaseDate: 2020-12-11
+    eol: 2021-12-31
+    lts: true
+    supportedJavaVersions: 8, 11
+    latest: "3.7.7"
+    latestReleaseDate: 2021-12-17
+
+-   releaseCycle: "3.6"
+    releaseDate: 2020-10-16
+    eol: 2020-10-16
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.6.0"
+    latestReleaseDate: 2020-10-16
+
+-   releaseCycle: "3.5"
+    releaseDate: 2020-08-31
+    eol: 2020-08-31
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.5.0"
+    latestReleaseDate: 2020-08-31
+
+-   releaseCycle: "3.4"
+    releaseDate: 2020-06-14
+    eol: 2021-06-30
+    lts: true
+    supportedJavaVersions: 8, 11
+    latest: "3.4.6"
+    latestReleaseDate: 2021-06-17
+
+-   releaseCycle: "3.3"
+    releaseDate: 2020-05-10
+    eol: 2020-05-10
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.3.0"
+    latestReleaseDate: 2020-05-10
+
+-   releaseCycle: "3.2"
+    releaseDate: 2020-04-01
+    eol: 2020-04-01
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.2.0"
+    latestReleaseDate: 2020-04-01
+
+-   releaseCycle: "3.1"
+    releaseDate: 2020-02-20
+    eol: 2020-02-20
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.1.0"
+    latestReleaseDate: 2020-02-20
+
+-   releaseCycle: "3.0"
+    releaseDate: 2019-11-24
+    eol: 2020-02-20
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.0.1"
+    latestReleaseDate: 2020-01-11
+
+-   releaseCycle: "2.25"
+    releaseDate: 2020-01-19
+    eol: 2022-01-31
+    lts: true
+    supportedJavaVersions: 8
+    latest: "2.25.4"
+    latestReleaseDate: 2021-05-24
+
+---
+
+> [Apache Camel](https://www.groovy-lang.org/) is an open source framework for message-oriented
+> middleware with a rule-based routing and mediation engine that provides a Java object-based
+> implementation of the Enterprise Integration Patterns using an application programming interface
+> (or declarative Java domain-specific language) to configure routing and mediation rules.
+
+Apache Camel follows [Semantic Versioning](https://semver.org/). From Camel 2.25 onward,
+the project has both LTS and non-LTS releases. There is two LTS releases per year, and they are
+supported for one year with security and important/critical bug fixes. Non-LTS releases
+[don't receive any support](https://camel.apache.org/blog/2020/03/LTS-Release-Schedule/), which is
+why they are marked EOL as soon as they are released.
+
+Camel have [the following Java requirements](https://groovy.apache.org/download.html):
```

Comment:
Why java requirements link showing groovy ?
This link might be much better : https://camel.apache.org/manual/what-are-the-dependencies.html

Reply:
Fixed, I used the Groovy page as a base and forgot to update the link.